### PR TITLE
[NeuroHackademy] Adds an R profile.

### PR DESCRIPTION
**Pull request by arokem**
Is this correct? I was not sure how to adjust the image just for one image, but I reasoned that maybe this is the way to over-ride the global setting?

### CODE REVIEW

**Comment by arokem** [comment]
Changed to WIP, because I will need to change the tag once I have the image correctly configured. I am still experimenting with that.

**Comment by arokem** [comment]
Removed the WIP designation, because I have now tested this and this image/tag should work fine for our purposes.

**Comment by yuvipanda** [comment]
I think it should work too - try it after this lands, we can tweak it monday if needed

**Comment by arokem** [comment]
Doesn't look like it works. I get a new profile, but it uses the same image as the regular CPU instance.

File: config/clusters/2i2c-aws-us/neurohackademy.values.yaml [hunk 1]
```diff
@@ -107,6 +107,12 @@ jupyterhub:
           extra_resource_limits:
             nvidia.com/gpu: "1"
           init_containers: *init_containers
+      - display_name: R machine
+        slug: R
+        description: "Start a container with R available"
+        image:
```
Comment (arokem) [inline config/clusters/2i2c-aws-us/neurohackademy.values.yaml:113]: Does this need to be nested in a `profile_options` block? That's my guess just based on reading the profile for the GPU instance.

Comment (yuvipanda) [reply config/clusters/2i2c-aws-us/neurohackademy.values.yaml:113]: Ah that is the correct diagnosis. I suppose this is the negative of looking at this while on my phone. Sorry for not catching it

Comment (arokem) [reply config/clusters/2i2c-aws-us/neurohackademy.values.yaml:113]: Thanks! See #4513

**Pull request merged**